source/reference/collection-level-filtering.txt

```diff
@@ -0,0 +1,203 @@
+and the ``mongosync_reserved_for_internal_use`` system database on the
+destination cluster. You must remove those databases before restarting
+``mongosync`` with your new filter.
+
+Follow these steps to prepare the destination cluster for a new filter.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Remove ``mongosync_reserved_for_internal_use``.
+
+      a. Stop the ``mongosync`` process. 
+      
+      #. Connect to the :binary:`mongos` instance in the destination
+         cluster with :binary:`mongosh`.
+
+      #. Drop the ``mongosync_reserved_for_internal_use`` system
+         database.
+
+         .. code-block:: javascript
+          
+            use mongosync_reserved_for_internal_use
+            db.dropDatabase()
+
+   .. step:: Remove user databases.
+
+      a. List the databases in the cluster
+
+         .. code-block:: javascript
+          
+            show databases
+
+      #. Remove user databases. The ``admin`` and ``config`` databases
+         are system databases. If the ``show databases`` command lists
+         any other databases on the destination cluster, you must remove
+         them.
+
+         Repeat this step for each user database list:
+
+         .. code-block:: javascript
+          
+            use <user database name>
+            db.dropDatabase()
+            db.dropDatabase()
+
+         Note: After the first :method:`db.dropDatabase()` operation
+         completes, you need to run it a second time to remove the
+         database.
+
+   .. step:: Configure a New Filter.
+
+      a. Decide which databases and collections you want to filter on.
+         Add the databases and collections to the ``includeNamespaces``
+         array. For configuration details, see
+         :ref:`c2c-configure-filter`.
+
+      #. Run ``mongosync`` to :ref:`reconnect <c2c-conn-top-level>` the
+         source and destination clusters.
+
+      #. Use the ``/start`` API end point to start syncing. Be sure to 
+         attach your new filter when you call ``/start``.
+
+.. _c2c-filter-with-out:
+
+Filtering with $out
+-------------------
+
+The :ref:`$out <agg-out>` aggregation stage creates a new collection
+when it runs. You can use the ``$out`` stage with filtering if you are
+filtering on the whole database and not just the collection specified in
+the ``$out`` statement.
+
+For example, consider this aggregation pipeline:
+
+.. code-block:: javascript
+
+      db.books.aggregate( [
+         { $group : { _id : "$author", books: { $push: "$title" } } },
+         { $out : "authors" }
+      ] )
+
+The ``$out`` stage creates the ``authors`` collection in the ``books``
+database. If you want to include the ``authors`` collection, you must
+specify the entire ``books`` database in your filter. The filter will
+not work if you only specify the ``authors`` collection.
+
+This filter works:
+
+.. code-block:: json
+
+   "includeNamespaces": [
+     {
+       "database": "books"
```

Comment:
I'd call the database literature or something, its confusing if both a collection and DB have the same name.